t/op/tr.t

```diff
@@ -1211,4 +1211,14 @@ for ("", nullrocow) {
     is $c, 1, "Count for the above test";
 }
 
+{   # As of August 2024, the code to handle this situation is not otherwise
+    # exercised by the test suite
+
+    my $A = "A";
+    utf8::upgrade($a);
```

Comment:
Why `$A` (upper-case) on line 1217, but `$a` (lower-case) on line 1218?

Side note: In general, IMO, we should avoid use of `$a` and `$b` because of their special characteristics with respect to the `sort` function.

Reply:
Why?  Because I screwed up.  And good point about `$a`